### DISCUSSION TITLE
[Feature Request] Set SoundDrivers key from Options

### DISCUSSION THREAD

**Discussion by ghost** [TeamRizu/OutFox#358] [03 - Feature Requests]
It's really hard to get the SoundDriver for PulseAudio working in OutFox. (To the novice user, that is.)

The steps you'd have to take are:
-Try hacky solutions
-Bang your head against the keyboard
-Do a grep of StepMania's code
-Finally come to the solution that is changing the SoundDrivers key in Save/Preferences.ini to Pulse

I believe a more user-friendly way to do this would be through setting the SoundDriver through the Options menu.
i.e.
Options -> Sound -> Audio Driver: (ALSA-sw, Pulse)

**Comment by Scraticus** [comment]
I like this idea, when we finish rewriting the drivers, i'll ensure it happens.
